packages/web-api/src/types/request/files.ts

```diff
@@ -44,7 +44,7 @@ interface FileChannelDestinationArgument {
 }
 interface FileThreadDestinationArgument {
   /** @description Channel ID where the file will be shared as a thread reply. */
-  channel_id: string;
+  channel_id?: string;
```

Comment:
`channel_id` is optional in `FileChannelDestinationArgument` and optional in `FileUploadV2Job`